gradle.properties

```diff
@@ -116,7 +116,7 @@ artifactRedirecting.androidx.annotation.version=1.9.1
 artifactRedirecting.androidx.graphics.version=1.0.1
 artifactRedirecting.androidx.lifecycle.version=2.9.0-alpha03
```

Comment:
About commit "Apply savedstate 1.3.0-alpha06 from AOSP."

What are your changes and what are the AOSP state? Better to separate these commits in the future.

Reply:
do you meant to have a commit with invalid state and then to fix it?

Reply:
yes

Reply:
done